examples/docs/en-US/tabs.md

```diff
@@ -84,7 +102,7 @@ Tabs styled as cards.
 
 Closable tabs.
 
-:::demo You can set `closable` attribute in `el-tabs`. It accept `Boolean` and Tab will be closable when the boolean is `true`.
+:::demo You can set `closable` attribute in `el-tabs` to make the tabs to be closable, `closable` is also can be setting in the tab panel to make partial tabs to be closable.
```

Comment:
You can set the `closable` attribute in `el-tabs` to make all tabs closable. Also, `closable` can be set in a tab panel to make that specific tab closable.